app/models/incomeSourceDetails/TaxYear.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails
+
+case class TaxYear(startYear: String, endYear: String)
```

Comment:
there was a few suggestion with respect to this model ( double check with @ayubAz  what is his plan to apply changes etc)
startYear / endYear - to be converted to Int and we might need to add 
required(startYear + 1 == endYear, "Incorrect taxYear definition ..") to definition